dev/Interop/StoragePickers/PickerLocalization.cpp

```diff
@@ -9,14 +9,47 @@
 #include "PickerLocalization.h"
 #include <winrt\base.h>
 #include <winrt\Microsoft.Windows.ApplicationModel.Resources.h>
-#include <iostream>
+#include <array>
+
+
+const winrt::hstring priFileName = L"Microsoft.WindowsAppRuntime.pri";
```

Comment:
Don't declare global constructor as it adds to load time perf hit for anyone loading the DLL, whether or not they ever using this functionality